ghcup-vanilla-0.0.8.yaml

```diff
@@ -4572,6 +4572,110 @@ ghcupDownloads:
+              dlHash: d2f2da6e93ceaa45a1c820720f75b0d06f624aed2377cc777366be8449dbd17f
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-centos7-linux.tar.xz
+            unknown_versioning: *ghc984-x86_64-centos7
+          Linux_Debian:
+            (>= 10 && < 11):
+              dlHash: 26bc72eb2758a3df14a4a88193d116d2cf334c2a20892b401f5cd3f82e6d346c
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-deb10-linux.tar.xz
+            < 10:
+              dlHash: 1ff17998daa7d356801899b2862916c8df53ae57969189419d7709a877fc380c
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-deb9-linux.tar.xz
+            '>= 11': &ghc984-x86_64-deb11
+              dlHash: af151db8682b8c763f5a44f960f65453d794c95b60f151abc82dbdefcbe6f8ad
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-deb11-linux.tar.xz
+            unknown_versioning: *ghc984-x86_64-deb11
+          Linux_Fedora:
+            '>= 33':
+              dlHash: 5f03d48f118abd30aee37d8bcddc1d7012193ff205b15121807ecd979c4cf947
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc984-x86_64-centos7
+          Linux_Mint:
+            < 20: &ghc984-x86_64-ubuntu1804
+              dlHash: e826990bd46a93ff8c87cf279fe1c480d763df4f1140fdf33b26d22ef3e21f06
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc984-x86_64-ubuntu2004
+              dlHash: d83941c9be977e7dc1a6f3eca32bc11e01a7cc3e6b678a226856a99e2ae22ce9
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            '>= 8': &ghc984-x86_64-rocky8
+              dlHash: 7e10f00bdc9e552d9fc3d330b27681b340b0653df4741983b49937c2b533a5c4
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-rocky8-linux.tar.xz
+            unknown_versioning: *ghc984-x86_64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc984-x86_64-ubuntu1804
+            unknown_versioning: *ghc984-x86_64-ubuntu2004
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc984-x86_64-rocky8
+          Windows:
+            unknown_versioning:
+              dlHash: 184c9184d6439f1da8bca0214011ef6074fb5946eaa5840dc0c3e884515d7c52
+              dlOutput: ghc-9.8.4-x86_64-windows.tar.xz
```

Comment:
delete

Reply:
This disagrees with the filename of `dlUri`; is this okay?

Reply:
The main point of `dlOutput` is to avoid collisions in case the `dlUri` is not unique enough.

Reply:
Okay, this is useful information.

The natural next question is then: how would one find this out? Is the format of this file documented somewhere?

Reply:
All normal bindists are unique, because they contain information on:

- version
- architecture
- platform
- distro

I have not seen an overlap in bindist names so far from GHC pipeline.

This was added for cross bindists afair that had no unique names.